<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineServicesPage</class>
 <widget class="QWidget" name="OnlineServicesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>120</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vlayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ItemView" name="view"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ToolButton" name="searchButton"/>
     </item>
     <item>
      <widget class="GenreCombo" name="genreCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MenuButton" name="menuButton"/>
     </item>
     <item>
      <widget class="ToolButton" name="addToPlayQueue"/>
     </item>
     <item>
      <widget class="ToolButton" name="replacePlayQueue"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemView</class>
   <extends>QTreeView</extends>
   <header>itemview.h</header>
  </customwidget>
  <customwidget>
   <class>GenreCombo</class>
   <extends>QComboBox</extends>
   <header>genrecombo.h</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>toolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MenuButton</class>
   <extends>QToolButton</extends>
   <header>menubutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
